miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java

```diff
@@ -110,22 +127,21 @@ public void setSecurityManager(SecurityManager securityManager) {
   public Collection<String> populatePlatformTypes() {
     return PlatformType.getKeys();
   }
-  
+
   public Boolean misoPropertyBoolean(String property) {
     MisoPropertyExporter exporter = (MisoPropertyExporter) ApplicationContextProvider.getApplicationContext().getBean("propertyConfigurer");
     Map<String, String> misoProperties = exporter.getResolvedProperties();
-    return misoProperties.containsKey(property)
-        && Boolean.parseBoolean(misoProperties.get(property));
+    return misoProperties.containsKey(property) && Boolean.parseBoolean(misoProperties.get(property));
   }
 
   @ModelAttribute("autoGenerateIdBarcodes")
   public Boolean autoGenerateIdentificationBarcodes() {
     return misoPropertyBoolean("miso.autoGenerateIdentificationBarcodes");
   }
-  
+
   @ModelAttribute("maxLengths")
   public Map<String, Integer> maxLengths() throws IOException {
-    return requestManager.getPoolColumnSizes();
+    return DbUtils.getColumnSizes(interfaceTemplate, "Pool");
```

Comment:
I'm guessing merge/rebase error. RequestManager method was the correct source for this. Controllers shouldn't require database knowledge (jdbc/table names)